app/assets/stylesheets/components/_button.sass

```diff
@@ -0,0 +1,19 @@
+$Button-default-backgroundColor: Theme-color(button, default)
+$Button-text-color: Theme-color(text, lighter)
+$Button-height: 50px
+$Button-spacing: $Theme-spacing-base
+
+.Button
```

Comment:
Yeah, it doesn't make sense... 
Although my idea was to have a button with no colors, only spacing border, and text alignment. Maybe this is a perfect example to use modifiers, but one thing i can't understand is where do i set the actual size of the button?